conf/test_netmhcii.config

```diff
@@ -4,20 +4,20 @@
  * -------------------------------------------------
  * Defines bundled input files and everything required
  * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/epitopeprediction -profile test_netmhcii,<docker/singularity>
+ *   nextflow run nf-core/epitopeprediction -profile test_netmhcii,<docker/singularity> --outdir <OUTDIR>
  */
 
 params {
-  config_profile_name = 'NetMHCII Test Profile'
-  config_profile_description = 'Peptide list based test profile for NetMHCII'
+    config_profile_name = 'NetMHCII Test Profile'
+    config_profile_description = 'Peptide list based test profile for NetMHCII'
 
-  max_cpus = 2
-  max_memory = 6.GB
-  max_time = 48.h
+    max_cpus = 2
+    max_memory = 6.GB
+    max_time = 48.h
 
-  // Input data
-  input = 'https://raw.githubusercontent.com/nf-core/test-datasets/epitopeprediction/testdata/sample_sheets/sample_sheet_peptides_class2.csv'
+    // Input data
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/epitopeprediction/testdata/sample_sheets/sample_sheet_peptides_class2.csv'
 
-  tools = 'netmhcii'
-  netmhcii_path = './non-free/netmhcii.tar.Z'
+    tools = 'netmhcii-2.2'
```

Comment:
We could also include here the most recent netmhcii verision right? That would be netmhcii-2.3